src/Task/Php/WebOpcacheResetCreateScriptTask.php

```diff
@@ -92,7 +92,7 @@ public function execute(Node $node, Application $application, Deployment $deploy
         $options = $this->configureOptions($options);
 
         $workspacePath = $deployment->getWorkspacePath($application);
-        $webDirectory = $application->hasOption('webDirectory') ? $application->getOption('webDirectory') : 'Web';
+        $webDirectory = $application->hasOption('webDirectory') ? $application->getOption('webDirectory') : 'public';
```

Comment:
Could we set the default based on application?

Reply:
I added all the defaults in the different applications

Reply:
As you can see before it was totally messy. Sometimes "web" was used, sometimes "Web"

Reply:
> As you can see before it was totally messy. Sometimes "web" was used, sometimes "Web"

It was no mess. The "Web" comes from the Flow. At this time, TYPO3 had no web folder. Then TYPO3 introduced "web" as default and now changed to "public" as default.

Reply:
so will you approve or do I need to change something?

Reply:
I would add a constant for the default public folder to each app and use it as fallback. Somethink like 
```
$webDirectory = $application->hasOption('webDirectory') ? $application->getOption('webDirectory') : $application::defaultPublicFolder;
```

Reply:
Is it ok to have the global Application default also set to 'public' or not?

Reply:
@simonschaufi You should also set a defaultPublicFolder constant for the default Application. Yes.